sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaRegistryClient.cs

```diff
@@ -234,6 +248,48 @@ await GetSchemaPropertiesInternalAsync(groupName, schemaName, schemaDefinition,
 #pragma warning restore AZC0015 // Unexpected client method return type.
             GetSchemaInternalAsync(schemaId, false, cancellationToken).EnsureCompleted();
 
+        /// <summary>
+        /// Gets the schema content associated with the group name, schema name, and version from the SchemaRegistry service.
+        /// </summary>
+        /// <param name="groupName"> Schema group under which schema is registered.  Group's serialization type should match the serialization type specified in the request.  </param>
+        /// <param name="schemaName"> Name of schema. </param>
+        /// <param name="schemaVersion"> Version number of specific schema. </param>
+        /// <param name="cancellationToken">The cancellation token for the operation.</param>
+        /// <returns>The properties of the schema, including the schema content provided by the service.</returns>
+#pragma warning disable AZC0015 // Unexpected client method return type.
+        public virtual Response<SchemaRegistrySchema> GetSchema(string groupName, string schemaName, int schemaVersion, CancellationToken cancellationToken = default) =>
+#pragma warning restore AZC0015 // Unexpected client method return type.
+            GetSchemaInternalAsync(groupName, schemaName, schemaVersion, false, cancellationToken).EnsureCompleted();
+
+        private async Task<Response<SchemaRegistrySchema>> GetSchemaInternalAsync(string groupName, string schemaName, int version, bool async, CancellationToken cancellationToken)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope(GetSchemaScopeName);
```

Comment:
We should align on the scope name used here across languages. I suspect other languages may be using "version" in the name because they have defined a separate method. Would you mind bringing this up in the Teams chat?

Reply:
Consensus as far as I could tell was to go with having version in the name

Reply:
Hmm I'm not sure this will work for us as there is tooling that checks that the name matches the method name.

Reply:
Though perhaps this validation is skipped when using ForwardClientCalls.

Reply:
If all the checks are passing does that mean that the validation is skipped? Or should we consider switching the method name to be `GetSchemaByVersion`?

Reply:
Updated the scope to reflect the method name based on the teams conversation

Reply:
Thanks for chasing this down!